opentelemetry/spring-boot-instrumentation/docker-compose.creds.yaml

```diff
@@ -12,12 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Use this compose file along with docker-compose.yaml to pass Application Default
-# Credentials from the host into the collector container:
+# Use this compose file along with docker-compose.yaml to pass a credentials file from the host
+# into the collector container:
 #
 # ```
-# export GOOGLE_APPLICATION_CREDENTIALS=$HOME/.config/gcloud/application_default_credentials.json
-# docker compose -f docker-compose.yaml -f docker-compose.adc.yaml up
+# GOOGLE_CLOUD_PROJECT="otel-quickstart-demos" \
+# GOOGLE_APPLICATION_CREDENTIALS="$HOME/.config/gcloud/application_default_credentials.json" \
```

Comment:
what is a reason to advertise the anti-pattern of using the GSA key file for authentication?
the doc samples use the main use case which is to run the sample on Google Cloud.

Reply:
This is to allow users to run the example on their own machine. We are not recommending this pattern in the docs, just making it possible.